codes/quantum/qudits_galois/galois_stabilizer.yml

```diff
@@ -10,18 +10,27 @@ logical: galois
 name: 'Galois-qudit stabilizer code'
 introduced: '\cite{arXiv:quant-ph/0005008}'
 
-description: 'An \(((n,p^k,d))_{GF(q)}\) stabilizer code is denoted as \([[n,k]]_{GF(q)}\) or \([[n,k,d]]_{GF(q)}\), where \(d\) is the code''s distance. Stub.'
-# Is it q^k?
-#'An \(((n,q^k,d))_{GF(q)}\) stabilizer code is denoted as \([[n,k]]_{GF(q)}\) or \([[n,k,d]]_{GF(q)}\), where \(d\) is the code''s distance.'
+description: 'An \(((n,q^k,d))_{GF(q)}\) stabilizer code is denoted as \([[n,k]]_{GF(q)}\) or \([[n,k,d]]_{GF(q)}\), where \(d\) is the code''s distance and \(q=p^m\).
+The {GF(\(q\))} stabilizer of \(Q\) is the set \(S(Q) = \{M \in P_n(q)|\, |\psi\rangle\) is a generalized eigenvector of  \(M\) with generalized eigenvalue 0 \(\forall |\psi\rangle \in Q\}  \).
```

Comment:
Why generalized e-vector and not just e-vector?

Reply:
Also, isn't the e-value +1? If you have time, please try to change format to more closely match stabilizer.yml and prime_qudit_stabilizer.yml

Reply:
I actually have the same confusion. This definition comes from Gottesman's textbook Definition 8.6 (page 124). 
I now adapt the definition of code_id: stabilizer to definite the Galois-qudit stabilizer. 
I also simply commented out the old definition so that you are free to delete it if you find it useless.